sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/model/FeedResponseDiagnostics.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos;
+package com.azure.cosmos.model;
```

Comment:
why FeedResponseDiagnostics is a model?

Reply:
I was not sure about this one. But since CosmosResponseDiagnostics is not a model too, so I agree, this shouldn't be either. Will move this back to cosmos package.